### ISSUE TITLE
Keeper hand is at an offset to system cursor

### ISSUE THREAD

**Issue by akien-mga** [OpenDungeons/OpenDungeons#1298]
- System: Mageia 7 x86_64
- Dependencies: Ogre 1.10.12 & 1.11.2, CEGUI 0.8-git (2018-03-22), OIS 1.4.0, SFML 2.5.0

Building the current master branch da9beaf (with Ogre 1.10.12) or #1291 (with Ogre 1.10.12 and 1.12.2), I get an offset between the system cursor and the keeper hand mesh which makes the cursor almost unusable.

![screenshot_20181109_101329](https://user-images.githubusercontent.com/4701338/48253650-1bad1780-e408-11e8-90ed-737d18422f49.png)

I have yet to investigate what causes it, I'll see if it's due to OIS (we used to build against 1.3.0).

**Comment by akien-mga** [comment]
There's actually a difference between Ogre 1.10.12 and 1.11.2:

- Ogre 1.10.12: System cursor has hotspot, keeper hand is at a big offset (above screenshot)
- Ogre 1.11.2: System cursor is located in palm of the keeper hand, and the keeper hand's fingertips have the hotspot (see how I point Multiplayer with system cursor but it's the fingertips which define Skirmish as hovered)
![screenshot_20181109_103756](https://user-images.githubusercontent.com/4701338/48254907-86ac1d80-e40b-11e8-95c4-88c36526a2b9.png)

**Comment by akien-mga** [comment]
Same behaviour with OIS 1.3 so it doesn't seem to be an OIS regression.

The above descriptions were for running the game in windowed mode on 1920x1080 - the Y offset seen with Ogre 1.11.2 seems to be equivalent to the window title bar + the system task bar (which eventually make the game viewport less than 1080p).

Running in fullscreen, the system cursor is at the correct hotspot position:
![screenshot_20181109_105610](https://user-images.githubusercontent.com/4701338/48255821-118e1780-e40e-11e8-97b7-990037c80bbd.png)

**Comment by akien-mga** [comment]
This seems to work properly also in windowed mode when building with `OD_USE_SFML_WINDOW=ON`, whether building against Ogre 1.10.12 or 1.11.2.

@oyvindln Should we go ahead and switch that on by default / eventually deprecate OIS?

**Comment by oyvindln** [comment]
Yeah it would be nice to deprecate IOS. Also nice to avoid one dependency. I think SFML windowing is mostly working fine on linux, don't remember if it's sorted on windows yet.
